js/embed_media_iframe_form.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Retrieve the name user input for embed media select iframe.
+ * Everytime the user goes to different page within the media embed select modal
+ * it will save the name field value to retrive the search result when user
+ * comes back to the first page.
+ */
+
+(function ($, Drupal) {
+    'use strict';
+
+    Drupal.behaviors.tide_media_media_form = {
+      attach: function (context, settings) {
+        let nameField = $('#views-exposed-form-tide-media-browser-media-browser input[name=name]')
+        const fromSubmit = $('#views-exposed-form-tide-media-browser-media-browser input[type=submit]')
```

Comment:
`formSubmit`?

Reply:
Why is one `let` and one `const`?

Reply:
1. here the `formSubmit` is view filter selection "Apply" button. So every time user hits the Apply button it checks the name field saves the user input value in the localStorage
2. it should be const, it is a mistake, fixing it now, thanks

Reply:
oops, fixing it :+1: